draft-ietf-httpbis-http2.xml

```diff
@@ -2601,11 +2601,12 @@ HTTP2-Settings    = token68
             received.
           </t>
           <t>
-            A <x:ref>SETTINGS</x:ref> frame can alter the initial flow control window size for all
-            streams in the "open" or "half closed (remote)" state. When the value of
-            <x:ref>SETTINGS_INITIAL_WINDOW_SIZE</x:ref> changes, a receiver MUST adjust the size of
-            all stream flow control windows that it maintains by the difference between the new
-            value and the old value.
+            In addition to changing the flow control window for future streams, a

```

Comment:
Note that I updated the alternative suggestion text above since the original post.